docs/concepts.rst

```diff
@@ -518,16 +518,14 @@ will invariably lead to block tasks that depend on their past successes.
 ``skipped`` states propagates where all directly upstream tasks are
 ``skipped``.
 
-If you want to skip some tasks, keep in mind that you can't have an empty
-path, if so make a dummy task.
+Note that when a path is a downstream task of the returned task (list), it will
+not be skipped:
 
-like this, the dummy task "branch_false" is skipped
+.. image:: img/branch_note.png
 
-.. image:: img/branch_good.png
-
-Not like this, where the join task is skipped
-
-.. image:: img/branch_bad.png
+Paths of the branching task are ``branch_a``, ``join`` and ``branch_b``. Since
+``join`` is a downstream task of ``branch_a``, it will be excluded from the skipped
+tasks when ``branch_a`` is selected by the Python function.
```

Comment:
_I would say is returned by the Python callable_ instead of _is selected by the Python function._

to keep using the same wording.

Reply:
Good point, fixed.